app/screens/channel_info/channel_info.js

```diff
@@ -178,7 +178,7 @@ export default class ChannelInfo extends PureComponent {
                     testID='channel_info.edit_channel.action'
                     theme={theme}
                 />
-                {callsFeatureEnabled &&
+                {isSupportedServerCalls &&
```

Comment:
shouldn't you check for `isCallsEnabled` as well? there is no need to mount the component if you already know that calls is not enabled for this channel

Reply:
As we discussed over a call, I added an on demand check for whether calls is enabled. If it isn't, we're changing the button text, and giving the user an alert:
<img width="444" alt="image" src="https://user-images.githubusercontent.com/1490756/160494823-011ec02f-adf2-4c04-8d43-c21cca20d798.png">